include/aspect/boundary_traction/initial_lithostatic_pressure.h

```diff
@@ -105,6 +105,12 @@ namespace aspect
          * based on depth interpolation between computed pressure values.
          */
         double interpolate_pressure (const Point<dim> &p) const;
+
+        /**
+         * A list of those boundary indicators that are prescribed
+         * the initial lithostatic pressure computed by this plugin.
+         */
+        std::set<types::boundary_id> traction_bi;
```

Comment:
Can you instead store a `bool prescribe_constant_pressure_at_bottom_boundary` and `types::boundary_id bottom_boundary_id`? This will allow optimizations I will explain below.